java/dagger/example/android/espresso/stubbed/StubNetworkingModule.java

```diff
@@ -0,0 +1,18 @@
+package dagger.example.android.espresso.stubbed;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.example.android.espresso.main.ApiClient;
+
+@Module
+public class StubNetworkingModule {
```

Comment:
Overriding modules is not recommended by the testing guide. https://google.github.io/dagger//testing.html